test/Dialect/LLHD/Transforms/unroll-loops.mlir

```diff
@@ -350,3 +352,50 @@ hw.module @DontCrashOnSingleBlocks() {
     llhd.yield
   }
 }
+
+// CHECK-LABEL: @DegenerateSingleTripLoopWithEq
+hw.module @DegenerateSingleTripLoopWithEq() {
+  %c0_i32 = hw.constant 0 : i32
+  %c1_i32 = hw.constant 1 : i32
+  llhd.combinational {
+    // CHECK: cf.br
+    // CHECK: func.call @marker()
+    // CHECK: cf.br
+    // CHECK: llhd.yield
```

Comment:
wouldn't that filecheck also succeed if the IR stays untouched? Maybe we need some additional CHECK-NOT?

Reply:
🤦 You're totally right! Let me adjust this a bit such that we can at least check the exact sequence of lines.